recipe/meta.yaml

```diff
@@ -28,22 +29,17 @@ requirements:
     - pywin32  # [win]
     - traitlets
 
-{% set skip_all = "not test_not_on_path and not test_path_priority" %}
-{% set skip_win_py2k = " and not test_migrate_config" %}
-{% set skip = skip_all %}  # [not(win and py2k)]
-{% set skip = skip_all ~ skip_win_py2k %}  # [win and py2k]
 test:
   requires:
-    # - ipykernel
-    - mock  # [py2k]
+    - ipykernel
     - nose
     - pytest
     - pip
   commands:
     - jupyter -h
     - jupyter-migrate -h
     - jupyter-troubleshoot --help
-    # - python -m pytest --pyargs jupyter_core -k "{{ skip }}"
+    - python -m pytest --pyargs jupyter_core
```

Comment:
:+1: i've also recently been adding `--pytest-cov` and a `--cov-fail-under` (set to whatever it is) just to backstop pip check missing a major possible test dependency...

Reply:
Great, thanks for the tips. I'm in unfamiliar territory here, so appreciate all the help!

Reply:
>  (set to whatever it is

Not sure "whatever it is" is. Do you want to submit a code suggestion, just to be clear?

I'm also fine to abort these testing changes and go back to what works...

Reply:
Sorry, I meant whatever (the threshold happens to be today) e.g. `76` on linux. This may well be a bridge too far, and can be added at another point. I like this as a prelude to turning on automerge, which otherwise gives me the willies for important packages. But you have the stuff working, and can do another day, if you'd rather we :shipit: !

Anyhoo: for posterity

```diff
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index f231d31..06168a2 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -29,19 +29,21 @@ requirements:
     - pywin32  # [win]
     - traitlets
 
-{% set skip = "not test_not_on_path and not test_path_priority" %}
+{% set skip = "not (not_on_path or path_priority)" %}
+
 test:
   requires:
     - ipykernel
     - nose
-    - pytest
     - pip
+    - pytest
+    - pytest-cov
   commands:
+    - pip check
     - jupyter -h
     - jupyter-migrate -h
     - jupyter-troubleshoot --help
-    - python -m pytest --pyargs jupyter_core -k "{{ skip }}"
-    - pip check
+    - python -m pytest -vv --pyargs jupyter_core -k "{{ skip }}" --cov jupyter_core --cov-fail-under 76
   imports:
     - jupyter_core
 

```